storage/mysql/log_storage_test.go

```diff
@@ -734,11 +752,25 @@ func TestGetTreeRevisionAtSize(t *testing.T) {
 		}
 
 		// But an intermediate value shouldn't work
-		treeRevision3, err := tx.GetTreeRevisionAtSize(21)
+		treeRevision3, err := tx.GetTreeRevisionAtSize(21, true)
 
 		if err == nil {
```

Comment:
Suggestion: use the if initialization statement for variables that don't need to live past a single check?

if treeRevision3, err := tx.GetTreeRevisionAtSize(21, true); err == nil {
    t.Fatalf("Unexpectedly returned revision for nonexistent tree size: %d", treeRevision3)
}

(Ditto for other occurrences.)